fixtures/parallelizer/__init__.py

```diff
@@ -251,7 +251,9 @@ def __init__(self, config):
             request = self.sprout_client.request_check(self.sprout_pool)
             self.appliances = []
             # Push an appliance to the stack to have proper reference for test collection
-            IPAppliance(address=request["appliances"][0]["ip_address"]).push()
+            # FIXME: this is a bad hack based on the need for controll of collection partitioning
```

Comment:
Once we resolve the test outcomes (is that on the agenda for the sprint) we will be in a position to "skip/something" those tests at runtime, rather than at collection